messaging/src/main/java/io/lantern/messaging/metadata/Metadata.kt

```diff
@@ -153,9 +153,12 @@ class Metadata(val mimeType: String?, val thumbnail: ByteArray?, val thumbnailMi
         }
 
         private fun audioMetadata(file: File, defaultMimeType: String): Metadata {
+            val duration = MediaMetadataRetriever().apply {
+                setDataSource(file.absolutePath)
+            }.extractMetadata(MediaMetadataRetriever.METADATA_KEY_DURATION)?.toDoubleOrNull()
```

Comment:
Extract the duration of the audio file, in case that such audio is corrupt or invalid the data is gonna be null